pages/deployment/[uuid].tsx

```diff
@@ -84,9 +96,9 @@ function CodeLine({ line }) {
 
 export default function Deployment() {
   const router = useRouter()
-  const { uuid }: { uuid?: string } = router.query
+  const { uuid, tab }: { uuid?: string; tab?: TabOptions } = router.query
 
-  const [tab, setTab] = useState<TabOptions>('overview')
+  const [group, setGroup] = useState<TabOptions>('overview')
```

Comment:
Personal preference, but instead of having to set state to update it each time we change tab, we could do the following:
```
  const group: TabOptions = useMemo(() => (tab && isTabOption(tab) ? tab : 'overview'), [tab])
```
Same for the other file.

Reply:
Because this is how we do it across the app I'd rather keep things consistent. However, if we think that this change is worth making across the whole app I'm happy to raise a ticket to get it all changed in the future.